organisations/tests/test_models.py

```diff
@@ -127,3 +130,68 @@ def test_user_str():
         email="",
     )
     assert user_without_name_and_email.__str__() == user_without_name_and_email.username
+
+
+@pytest.mark.django_db
+def test_too_old_personal_data():
+    too_old = timezone.now() - relativedelta(months=24, days=1)
+    still_valid = timezone.now() - relativedelta(months=24, days=-1)
+
+    # ### EXPECTED PERSON OBJECTS IN RESULTS
+
+    expected_people = PersonFactory.create_batch(5, user=None)
```

Comment:
trivial, but it would be better to "name" the persons here where the variable is defined, instead of using the indexes.

```suggestion
    old_enrolment_person, old_study_group_person, multi_enrolment_person, shared_all_old_person  = PersonFactory.create_batch(5, user=None)
```

Reply:
Good idea, changed.